frontend/public/locales/en/translation.json

```diff
@@ -332,6 +332,7 @@
   "A project organizes all of your Google Cloud resources. A project consists of a set of users; a set of APIs; and billing, authentication, and monitoring settings for those APIs. So, for example, all of your Cloud Storage buckets and objects, along with user permissions for accessing them, reside in a project.": "A project organizes all of your Google Cloud resources. A project consists of a set of users; a set of APIs; and billing, authentication, and monitoring settings for those APIs. So, for example, all of your Cloud Storage buckets and objects, along with user permissions for accessing them, reside in a project.",
   "A proxy URL to use for creating HTTP connections outside the cluster. The URL scheme must be HTTP.": "A proxy URL to use for creating HTTP connections outside the cluster. The URL scheme must be HTTP.",
   "A proxy URL to use for creating HTTPS connections outside the cluster. If this is not specified, then httpProxy is used for both HTTP and HTTPS connections.": "A proxy URL to use for creating HTTPS connections outside the cluster. If this is not specified, then httpProxy is used for both HTTP and HTTPS connections.",
+  "A Red Hat OpenShift cluster that is hosted on the Red Hat OpenShift Container Platform in your on-premise datacenter.": "A Red Hat OpenShift cluster that is hosted on the Red Hat OpenShift Container Platform in your on-premise datacenter.",
```

Comment:
Small thing, should be data center instead, from https://issues.redhat.com/browse/ACM-4928

Reply:
Oops - I updated in the code but forgot to update translations. Pushing.